x-pack/plugins/apm/server/lib/transactions/get_transaction.ts

```diff
@@ -58,15 +54,5 @@ export async function getTransaction(
   }
 
   const resp: SearchResponse<Transaction> = await client('search', params);
-  const result = oc(resp).hits.hits[0]._source();
-
-  if (result === undefined) {
-    const notFoundError = new Error(
-      `No results found for transaction ID ${transactionId} and trace ID ${traceId}`
-    ) as HttpError;
-    notFoundError.statusCode = 404;
```

Comment:
I think we probably need to look at the API layer and think about how to handle errors, but I'm also a bit confused/intrigued as of this morning around how our queries work re: time ranges, so in light of that, removing 404 for now is fine with me.